src/coldcard.test.js

```diff
@@ -0,0 +1,149 @@
+import {ColdcardMultisigWalletConfig, WALLET_CONFIG_VERSION} from './coldcard';
+
+describe("ColdcardMultisigWalletConfig", () => {
+  const jsonConfigUUID = {
+    name: "Test",
+    addressType: "P2SH",
+    network: "testnet",
+    quorum: {
+      requiredSigners: 2,
+      totalSigners: 3
+    },
+    startingAddressIndex: 5,
+    extendedPublicKeys: [
+      {
+        name: "unchained",
+        xpub: "tpubDF17mBZYUi35r7UEkGr7SjSisec8QL2J1zGh9WQtmnhJtHeMFy2eH2Xsnr2ynxENbqHmcEA4NnoT8T6RZxks4G5evZdWy1RbSPTm8LtNPU3",
+        bip32Path: "Unknown",
+        xfp: "83471e78"
+      },
+      {
+        name: "Os_words1",
+        xpub: "tpubDF61GHbPYRhEEsqNTDF2jbMBkEMmoksy1h2URbhZ8p7JfR9QRgbn6vkeA7g3t4Ue6uSHhYJxD9mRVz1ZQVYyW3RAPPuwVM4UeZyZPKu89DY",
+        bip32Path: "m/45'/1'/50'/0",
+        xfp: "f57ec65d"
+      },
+      {
+        name: "Os_words2",
+        xpub: "tpubDEZxsxeamoPim6T1hg2igm4vDmdQrQnHVH5gM5NjcjowtgYVv5ZhhR5sFgRVNjRFGk1HmxsFsYhu3jGaAGVCpCsL5AbAVk6xKssr6gK3tPk",
+        bip32Path: "m/45'/1'/60'/0",
+        xfp: "f57ec65d"
+      }
+    ],
+     uuid: "OWPyFOA1"
+  };
+  const coldcardConfigUUID = `# Coldcard Multisig setup file (exported from unchained-wallets)
```

Comment:
This doesn't need to be done in this PR necessarily, but I wonder if these would be useful in a fixtures file/directory? Might be something that's used later in other tests. Wouldn't even be surprised if there were some in the coldcard repo, which would be good to standardize on.

Not a blocking comment, just something that occurred to me looking at this again.

Reply:
Yes I think updating the fixtures in `unchained-bitcoin` with PSBT and signed PSBT will be necessary and improve confidence with known transactions, potentially even add it to the test suite.

Reply:
Would you mind adding this as an issue ticket to unchained-bitcoin and mentioning what specifically needs to be moved over? It might be best once this is merged in so there's something to link to but just wanted to mention here so we don't forget about it.

Reply:
I'd even support starting by adding a fixtures file here that we can eventually migrate to unchained-bitcoin when psbt support gets better built out there. Again, it shouldn't be a blocker necessarily, but I could see this being perpetually de-prioritized and more likely to have more constants that should be in fixtures added in as we continue building out.